editor/js/components/arcontroller.component.js

```diff
@@ -153,32 +153,37 @@ ArControllerComponent.prototype.startAR = function() {
                     vw = stream.videoWidth;
                     vh = stream.videoHeight;
 
-                    if(this.canvas.length==1)
-                    {
-                        //View page is the player
-                        var selectedCanvas = $(canvas[0]);
-                        selectedCanvas.before(stream);
-                        cw = selectedCanvas.width();
-                        ch = selectedCanvas.height();
-                        selectedCanvas.css("z-index",99);
-                        selectedCanvas.css("position","absolute");
-                    }
-                    else if(this.canvas.length>1)
-                    {
-                        //View page is the editor.
-                        var gameTab = $("#ingametab");
-                        gameTab.append(stream);
+                    if(this.canvas==null){
+                      console.log('canvas does not exist!\n');
```

Comment:
@ThorstenBux maybe we can find another solution here

Reply:
with https://github.com/augmentmy-world/arStudio/pull/19/commits/6f650e59412143fa358debc77ef35d006deb1665 i removed the console.log otherwise we receive too messages in the console.